src/neuroglancer/layer_dialog.ts

```diff
@@ -214,7 +214,9 @@ export class LayerDialog extends Overlay {
     this.setInfo('Validating volume source...');
     const token = this.volumeCancellationSource = new CancellationTokenSource();
     this.manager.dataSourceProvider
-        .getVolume(this.manager.chunkManager, url, /*options=*/undefined, token)
+        .getVolume(
+            this.manager.chunkManager, url,
+            /*options=*/{dataSourceProvider: this.manager.dataSourceProvider}, token)
```

Comment:
I think it would make sense to move this logic of passing through the dataSourceProvider to the getVolume, etc. functions in neuroglancer/datasource/index.ts that way it doesn't have to be duplicated.